proxy/src/main/java/com/velocitypowered/proxy/server/VelocityRegisteredServer.java

```diff
@@ -108,7 +116,8 @@ protected void initChannel(Channel ch) throws Exception {
             ch.pipeline()
                 .addLast(FRAME_DECODER, new MinecraftVarintFrameDecoder())
                 .addLast(READ_TIMEOUT,
-                    new ReadTimeoutHandler(server.getConfiguration().getReadTimeout(),
+                    new ReadTimeoutHandler(timeout < 0
```

Comment:
-1 usually means "no timeout", could we use 0 for the default value in the `PingOptions` class?

Reply:
In the end, some effective timeout is placed by the network equipment. I was noting that the value -1 is usually used to mean "no limit", whereas 0 sometimes means "use a sane default".

Reply:
Ok, I removed the limits for the timeout and documented the meaning of a value of 0 and a negative value